modules/monitoring-default-monitoring-components.adoc

```diff
@@ -29,14 +29,18 @@ By default, the {product-title} {product-version} monitoring stack includes thes
 |Alertmanager
 |The Alertmanager service handles alerts received from Prometheus. Alertmanager is also responsible for sending the alerts to external notification systems.
 
-|`kube-state-metrics` agent
-|The `kube-state-metrics` exporter agent (KSM in the preceding diagram) converts Kubernetes objects to metrics that Prometheus can use.
+|kube-state-metrics agent
+|The kube-state-metrics exporter agent (KSM in the preceding diagram) converts Kubernetes objects to metrics that Prometheus can use.
 
-|`openshift-state-metrics` agent
-|The `openshift-state-metrics` exporter (OSM in the preceding diagram) expands upon `kube-state-metrics` by adding metrics for {product-title}-specific resources.
+|monitoring-plugin
+|The monitoring-plugin dynamic plugin component deploys the monitoring pages in the *Observe* section of the {product-title} web console. 
+You can use Cluster Monitoring Operator (CMO) config map settings to manage monitoring-plugin resources for the web console pages.
```

Comment:
Note for future - maybe consider adding an attribute for the Operator name?

Reply:
> Note for future - maybe consider adding an attribute for the Operator name?

Thanks. Good idea. I'll write up a Jira for this.